content/authorization.md

```diff
@@ -1054,15 +1054,30 @@ preceding documentation.
   such devices to participate in the authorization
   ecosystem.
 
+- What happens when a user revokes my application's
+  access to their data?
+
+> When a user revokes an application's access, its
+  *refresh* tokens immediately become non-functional.
```

Comment:
```
*refresh* tokens immediately
```
Are there multiple refresh tokens? Or, will there only be one?

Wondering if we can drop the **s**.

Same question applies to the one below as well.

```
*Access* tokens cannot be 
```

Reply:
I think an application could have several in theory (though it's probably rare in practice).